tests/src/test/java/stepDefinitions/frontendstepdefinitions/LandingPageTest.java

```diff
@@ -1,48 +1,101 @@
 package stepDefinitions.frontendstepdefinitions;
 
-import ca.bc.gov.open.jagefilingapi.qa.config.ReadConfig;
+import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.EfilingPage;
 import ca.bc.gov.open.jagefilingapi.qa.frontend.pages.LandingPage;
 import ca.bc.gov.open.jagefilingapi.qa.frontendutils.DriverClass;
-import io.cucumber.java.en.And;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
 import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
 import org.junit.Assert;
+
 import java.io.IOException;
 
 public class LandingPageTest extends DriverClass {
 
-    ReadConfig readConfig;
-    LandingPage landingPage;
+     LandingPage landingPage;
+     EfilingPage eFilingPage;
 
-    @Given("browser is initialized")
-    public void browser_is_initialized() throws IOException {
+    @Before ("@frontend")
+    public void setUp() throws IOException {
         initializeDriver();
-        log.info("Browser is initialized");
+        log.info("Browser is initialized from the driver class");
     }
 
-    @Given("user is navigated to the landing page")
-    public void user_is_navigated_to_the_landing_page() throws IOException {
-        readConfig = new ReadConfig();
-
-        String url = readConfig.getBaseUrl();
-        driver.get(url);
-        log.info("Page is launched");
+    @After ("@frontend")
+    public void tearDown() {
+        driver.close();
+        driver.quit();
+        log.info("Browser closed");
     }
 
-    @When("page title is verified")
-    public void page_title_is_verified() {
+    @Given("user is on the landing page")
+    public void user_is_on_the_landing_page() throws IOException {
         landingPage = new LandingPage(driver);
 
+        landingPage.getUrl();
+        log.info("Landing page url is accessed successfully");
+
         String actualTitle = landingPage.verifyLandingPageTitle();
+        String expectedTitle = "eFiling Demo Client";
+
+        Assert.assertEquals(expectedTitle, actualTitle);
```

Comment:
should you use Junit 5 ?

Reply:
Sure I can. Just went with the latest version 6. Will downgrade. Are there any advantages with 5?